pkg/api/kptfile/v1alpha2/validation.go

```diff
@@ -27,8 +28,10 @@ const (
 )
 
 func (kf *KptFile) Validate() error {
+	const op errors.Op = "kptfile.validate"
 	if err := kf.Pipeline.validate(); err != nil {
-		return fmt.Errorf("pipeline is not valid: %w", err)
+		err =  fmt.Errorf("pipeline is not valid: %w", err)
+		return errors.E(op, err)
```

Comment:
Can we change this to `return errors.E(op, fmt.Errorf("pipeline is not valid: %w", err))`?

Reply:
done